song-client/src/test/java/org/icgc/dcc/song/client/SpringAppTest.java

```diff
@@ -40,7 +33,31 @@ public void testUpload(){
   @SneakyThrows
   @Ignore
   public void testStatus(){
-    executeMain("status", "-u", "UP-8b871d42-f86e-458d-8e4d-7e6688c597f9");
+    executeMain("status", "-u", "UP-826f10c8-7933-4160-9530-d24de40890ba");
+    log.info("sdf");
+  }
+
+  @Test
+  @SneakyThrows
+  @Ignore
+  public void testSave(){
+    executeMain("save", "-u", "UP-826f10c8-7933-4160-9530-d24de40890ba");
```

Comment:
Can you explain what the purpose of all these tests are? They don't seem to provide any benefit over just having a run profile for each command in your IDE.

Reply:
i use them to debug the client when something doesnt work as expected, but i could have also alternatively add a run profile for eachone. Should i remove all but one (as a placeholder) or remove the entire test?

Reply:
You can leave a single placeholder, remove the rest.